Microsoft.Toolkit.Uwp.UI.Controls.Core/TabbedCommandBar/TabbedCommandBar.cs

```diff
@@ -36,8 +37,10 @@ public class TabbedCommandBar : NavigationView
         public TabbedCommandBar()
         {
             DefaultStyleKey = typeof(TabbedCommandBar);
+            DefaultStyleResourceUri = new System.Uri("ms-appx:///Microsoft.Toolkit.Uwp.UI.Controls.Core/Themes/Generic.xaml");
```

Comment:
Tried to remove the need for this line of code, but was unsuccessful

Reply:
Yeah, we had this same issue with `ColorPicker` as well. We have an issue on WinUI for this here: https://github.com/microsoft/microsoft-ui-xaml/issues/3502 - I suppose we could link to it, but I think we need it on all controls for WinUI 3, right @azchohfi?